.github/workflows/release_deb.yml

```diff
@@ -0,0 +1,112 @@
+          python-version: 3.7
+      
+      - name: Restore Qt from cache
+        id: cache-qt
+        uses: actions/cache@v1
+        with:
+          path: ../Qt
+          key: Qt-ubuntu-16.04-${{ env.QT_VERSION }}
+
+      - name: Set up Qt environment
+        uses: jurplel/install-qt-action@v2
+        with:
+          cached: ${{ steps.cache-qt.outputs.cache-hit }}
+          version: ${{ env.QT_VERSION }}
+
+      - name: Build
+        run: |
+          mkdir build
+          cd build
+          cmake .. -DCMAKE_BUILD_TYPE=Release
+          cmake --build . --config Release -j2
+          
+      - name: Deploy
+        run: |
+          sudo apt install libxcb*
+          cd build/
+          wget -c "https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage"
+          chmod a+x linuxdeployqt*.AppImage
+          ./linuxdeployqt*.AppImage ./cpeditor -qmake=../../Qt/${{ env.QT_VERSION }}/gcc_64/bin/qmake
+          
+      - name: Preparing application files' directory
+        run: |
+          mkdir cpeditor_${VERSION}_amd64 && cd cpeditor_${VERSION}_amd64
+          mkdir usr && cd usr
+          mkdir share && cd share
+          mkdir applications && mkdir cpeditor && mkdir cpeditor/bin && mkdir doc && mkdir doc/cpeditor && mkdir icons
+          mv /home/runner/work/cpeditor/cpeditor/.ci/DEB/cpeditor.desktop ./applications
+          mv /home/runner/work/cpeditor/cpeditor/.ci/DEB/cpeditor.sh ./cpeditor/
+          mv /home/runner/work/cpeditor/cpeditor/build/plugins/ ./cpeditor
+          mv /home/runner/work/cpeditor/cpeditor/build/cpeditor ./cpeditor/bin/
+          mv /home/runner/work/cpeditor/cpeditor/.ci/DEB/copyright ./doc/cpeditor/
+          mv /home/runner/work/cpeditor/cpeditor/.ci/linux/cpeditor-icon.png ./icons/cpeditor.png
+          mv /home/runner/work/cpeditor/cpeditor/build/translations/ ./cpeditor
+          cd ../
+          mkdir lib && cd lib
+          mkdir cpeditor
+          mv /home/runner/work/cpeditor/cpeditor/build/lib/* ./cpeditor
+      
+      - name: Preapiring and setting DEBIAN version control directory
+        run: |
+          cd cpeditor_${VERSION}_amd64
+          mv /home/runner/work/cpeditor/cpeditor/.ci/DEB/DEBIAN .
+          wget -c "http://archive.ubuntu.com/ubuntu/pool/universe/h/hashdeep/md5deep_4.4-4_all.deb"
+          wget -c "http://archive.ubuntu.com/ubuntu/pool/universe/h/hashdeep/hashdeep_4.4-4_amd64.deb"
+          sudo apt install ./*.deb
+          rm *.deb
+          md5deep -rl usr/ > ./DEBIAN/md5sums
+          cd DEBIAN
+          chmod 775 p*
+          sed -i "s/\${ver}/$VERSION/" control
+          SIZE=$(du -s ../../cpeditor_${VERSION}_amd64/ | cut -f1)
+          sed -i "s/\${size}/${SIZE}/" control
+
+      - name: Build .deb
+        run: |
+          cd /home/runner/work/cpeditor/cpeditor/
+          fakeroot dpkg-deb --build cpeditor_${VERSION}_amd64
+          
+      - name: Release .deb
```

Comment:
I think you can release this deb with the current existing release.yml. Just add a new Matrix variable and copy paste steps by adding condition with `if`. What's your opinion @ouuan ?